set.mm

```diff
@@ -40489,8 +40585,20 @@ among classes ( ~ eq0 , as opposed to the weaker uniqueness among sets,
   ${
     abf.1 $e |- -. ph $.
     $( A class abstraction determined by a false formula is empty.
-       (Contributed by NM, 20-Jan-2012.) $)
+       (Contributed by NM, 20-Jan-2012.)  Avoid ~ ax-8 , ~ ax-10 , ~ ax-11 ,
+       ~ ax-12 .  (Revised by Gino Giotto, 28-Jun-2024.) $)
     abf $p |- { x | ph } = (/) $=
+      ( cab wfal c0 bifal abbii cdif cv wcel wn wa df-nul df-dif pm3.24 3eqtrri
+      cvv eqtri ) ABDEBDZFAEBACGHFRRIBJRKZUALMZBDTNBRROUBEBUBUAPGHQS $.
+    $( $j usage 'abf' avoids 'ax-8' 'ax-10' 'ax-11' 'ax-12'; $)
+  $}
+
+  ${
+    abfOLD.1 $e |- -. ph $.
+    $( Obsolete version of ~ abf as of 28-Jun-2024.  (Contributed by NM,
+       20-Jan-2012.)  (Proof modification is discouraged.)
+       (New usage is discouraged.) $)
+    abfOLD $p |- { x | ph } = (/) $=
       ( cab c0 wceq wn ab0 mpgbir ) ABDEFAGBABHCI $.
   $}
```

Comment:
This theorem/proof should also be kept because of its simplictiy. It is somewhat strange that the new proof uses ~df-dif.

Reply:
It's probably because of the definition of the empty set: `(/) = ( _V \ _V )`

Reply:
I provided a shorter proof that does not require ~df-dif. 2f2967a